draft-irtf-cfrg-vdaf.md

```diff
@@ -4603,8 +4611,15 @@ perhaps unlikely) for a large set of non-heavy-hitter values to share a common
 prefix, which would be leaked by a prefix tree with a sufficiently small
 threshold.
 
-The only known, general-purpose approach to mitigating this leakage is via
-differential privacy.
+A malicious adversary controlling the Collector and one of the Aggregators can
+further turn arbitrary non-heavy prefixes into heavy ones by tampering with the
+IDPF output at any position. While our construction ensures that the sum of all
+nodes evaluated at a level cannot exceed the value at the parent nodes, this
+still may allow an adversary to discover individual non-heavy strings.
```

Comment:
This is incorrect since we backed out passing the previous aggregate result to `is_valid()`.

Reply:
Fixed.